sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -85,6 +85,7 @@ COMMON_DEPEND=">=sys-apps/util-linux-2.30:0=[${MULTILIB_USEDEP}]
 	xkb? ( >=x11-libs/libxkbcommon-0.4.1:0= )"
 
 RDEPEND="${COMMON_DEPEND}
+	acct-group/render
```

Comment:
We already create the `render` group in the post pkg installation step. We did not add the acct-* packages from Gentoo upstream in the systemd ebuild was that would be more added maintenance for us.

Reply:
So we only need to add the baselayout change for initramfs? Could you please point to the sources where `render` gets added?

Reply:
Here is the link to the sources: https://github.com/flatcar-linux/coreos-overlay/blob/main/sys-apps/systemd/systemd-9999.ebuild#L536

I checked the acct-group, and it seems they run the commands in pre installation: https://github.com/gentoo/gentoo/blob/master/eclass/acct-group.eclass#L122-L127

Reply:
Thank you for the reference. I'll close my [PR for `acct-group/render`](https://github.com/flatcar-linux/portage-stable/pull/98) and update this PR. I'll also use the opportunity to define a GID for `render` and update my [baselayout PR](https://github.com/flatcar-linux/baselayout/pull/5) accordingly.